### ISSUE TITLE
"Unexpected closing" error report

### ISSUE THREAD

**Issue by carnendil** [jendrikseipp/rednotebook#436]
### Versions

RedNotebook version (make sure you have the latest version): 2.11 (``rednotebook.2.11.0~201903261902~ubuntu16.04.1`` from rednotebook stable PPA)

Operating system and version: Ubuntu 16.04

### Expected behavior
Jump between entries without error report of "unexpected closure" ("closing?").

### Actual behavior
While working, an Ubuntu error report window appears, indicating that RedNotebook has unexpectedly been closed.

### Steps to reproduce the behavior
When clicking on any date of the incorporated calendar and/or when using the Alt+Home shortcut to open the current day's entry, but not in every instance of performing such actions.

I'm not even completely sure this is the only way to trigger the error, or if it is random and _appears_ to be linked to this action. Nor does it happens rigth away if you just try jumping between dates. 

**But it occurs within minutes of using the program**, always. It has started happening since the last two updates from the PPA.

An error report pops up informing of an unexpected closing of the program, yet the application remains open and working. I've kept working and saving (automatically and manually) all data, but eventually a new error report will flash.

If I click on "continue" on the error report window, the program will launch again on a new window, without the first one having been closed at any point.

If I click on "continue" _after_ having closed the original window, a new instance of the program will not be launched. This was the case from which the log is pasted below.

### Log output when reproducing behavior

Please paste contents from $HOME/.rednotebook/rednotebook.log,

```
2019-04-04 12:25:37,039 INFO     Writing log to file "/home/rainer/.rednotebook/rednotebook.log"
2019-04-04 12:25:37,040 INFO     System encoding: utf-8
2019-04-04 12:25:37,040 INFO     Language code: es_NI
2019-04-04 12:25:40,388 INFO     Running in portable mode: False
2019-04-04 12:25:40,388 INFO     First Start: False
2019-04-04 12:25:40,388 INFO     RedNotebook version: 2.11
2019-04-04 12:25:40,402 INFO     System info: machine: x86_64, platform: Linux-4.15.0-47-generic-x86_64-with-Ubuntu-16.04-xenial, processor: x86_64, python_version: 3.5.2, release: 4.15.0-47-generic, system: Linux, GTK: (3, 18, 9), Glib: (2, 47, 3), PyGObject: (3, 20, 0), YAML: 3.11
2019-04-04 12:25:41,410 DEBUG    Default font: Ubuntu 11
2019-04-04 12:25:41,411 DEBUG    Default size: 11.0
2019-04-04 12:25:42,222 INFO     Cloud ignore list: ["\\'\\'<span", 'style\\=\\"background\\-color', 'style\\=\\"font\\-family', 'style\\=\\"color', "\\'\\'<kbd", 'style\\=\\"background', "\\'fontawesome\\'", 'style\\=\\"text\\-align', 'style\\=\\"font\\-size', '007b63\\', '\\"\\>\\&\\#x231b\\', "</span\\>\\'\\'", '007b63\\', '\\"\\>⌛</span\\>\\\'\\\'', 'title\\=\\"pdf\\"\\>\\&\\#xf1c1\\', "</span\\>\\'\\'", 'padding', 'font\\-size', 'absolute', 'bottom', 'position', 'relative\\', '\\"\\>c<small', 'relative\\', '\\"\\>c', '0\\.7em', 'style\\=\\"position', 'relative\\', '\\"\\>c', '0\\.3em', 'title\\=\\"pdf\\"\\>\\\uf1c1\\\'\\\'', 'relative\\', '\\"\\>c', '007b63\\', '\\"\\>\\⌛\\\'\\\'', 'title\\=\\"pdf\\"\\>\\&\\#xf1c1\\', "\\<\\/span\\>\\'\\'"]
2019-04-04 12:25:42,223 INFO     Cloud include list: ['gir', 'igr', 'ncc', 'rnc', 'rrd', 'sig']
2019-04-04 12:25:42,223 DEBUG    Start compiling regexes
2019-04-04 12:25:42,230 DEBUG    Finished
2019-04-04 12:25:42,336 DEBUG    Tray icon visible: False
2019-04-04 12:25:42,358 INFO     Opening journal at '/home/rainer/.rednotebook/data'
2019-04-04 12:25:42,358 DEBUG    Starting to load files in dir "/home/rainer/.rednotebook/data"
2019-04-04 12:25:42,359 DEBUG    Loading file "/home/rainer/.rednotebook/data/2010-07.txt"
2019-04-04 12:25:42,387 DEBUG    Loading file "/home/rainer/.rednotebook/data/2010-08.txt"
2019-04-04 12:25:42,420 DEBUG    Loading file "/home/rainer/.rednotebook/data/2010-09.txt"
2019-04-04 12:25:42,433 DEBUG    Loading file "/home/rainer/.rednotebook/data/2010-10.txt"
2019-04-04 12:25:42,456 DEBUG    Loading file "/home/rainer/.rednotebook/data/2011-02.txt"
2019-04-04 12:25:42,468 DEBUG    Loading file "/home/rainer/.rednotebook/data/2011-11.txt"
2019-04-04 12:25:42,480 DEBUG    Loading file "/home/rainer/.rednotebook/data/2011-12.txt"
2019-04-04 12:25:42,494 DEBUG    Loading file "/home/rainer/.rednotebook/data/2012-01.txt"
2019-04-04 12:25:42,504 DEBUG    Loading file "/home/rainer/.rednotebook/data/2012-03.txt"
2019-04-04 12:25:42,531 DEBUG    Loading file "/home/rainer/.rednotebook/data/2012-04.txt"
2019-04-04 12:25:42,539 DEBUG    Loading file "/home/rainer/.rednotebook/data/2012-06.txt"
2019-04-04 12:25:42,557 DEBUG    Loading file "/home/rainer/.rednotebook/data/2012-10.txt"
2019-04-04 12:25:42,567 DEBUG    Loading file "/home/rainer/.rednotebook/data/2012-11.txt"
2019-04-04 12:25:42,580 DEBUG    Loading file "/home/rainer/.rednotebook/data/2013-01.txt"
2019-04-04 12:25:42,593 DEBUG    Loading file "/home/rainer/.rednotebook/data/2013-02.txt"
2019-04-04 12:25:42,599 DEBUG    Loading file "/home/rainer/.rednotebook/data/2013-03.txt"
2019-04-04 12:25:42,607 DEBUG    Loading file "/home/rainer/.rednotebook/data/2013-04.txt"
2019-04-04 12:25:42,624 DEBUG    Loading file "/home/rainer/.rednotebook/data/2013-05.txt"
2019-04-04 12:25:42,638 DEBUG    Loading file "/home/rainer/.rednotebook/data/2013-06.txt"
2019-04-04 12:25:42,660 DEBUG    Loading file "/home/rainer/.rednotebook/data/2013-07.txt"
2019-04-04 12:25:42,701 DEBUG    Loading file "/home/rainer/.rednotebook/data/2013-08.txt"
2019-04-04 12:25:42,720 DEBUG    Loading file "/home/rainer/.rednotebook/data/2013-09.txt"
2019-04-04 12:25:42,733 DEBUG    Loading file "/home/rainer/.rednotebook/data/2013-10.txt"
2019-04-04 12:25:42,742 DEBUG    Loading file "/home/rainer/.rednotebook/data/2013-11.txt"
2019-04-04 12:25:42,757 DEBUG    Loading file "/home/rainer/.rednotebook/data/2013-12.txt"
2019-04-04 12:25:42,774 DEBUG    Loading file "/home/rainer/.rednotebook/data/2014-01.txt"
2019-04-04 12:25:42,798 DEBUG    Loading file "/home/rainer/.rednotebook/data/2014-02.txt"
2019-04-04 12:25:42,832 DEBUG    Loading file "/home/rainer/.rednotebook/data/2014-03.txt"
2019-04-04 12:25:42,854 DEBUG    Loading file "/home/rainer/.rednotebook/data/2014-04.txt"
2019-04-04 12:25:42,881 DEBUG    Loading file "/home/rainer/.rednotebook/data/2014-05.txt"
2019-04-04 12:25:42,901 DEBUG    Loading file "/home/rainer/.rednotebook/data/2014-06.txt"
2019-04-04 12:25:42,915 DEBUG    Loading file "/home/rainer/.rednotebook/data/2014-07.txt"
2019-04-04 12:25:42,929 DEBUG    Loading file "/home/rainer/.rednotebook/data/2014-08.txt"
2019-04-04 12:25:42,940 DEBUG    Loading file "/home/rainer/.rednotebook/data/2014-09.txt"
2019-04-04 12:25:42,941 DEBUG    Loading file "/home/rainer/.rednotebook/data/2014-10.txt"
2019-04-04 12:25:42,956 DEBUG    Loading file "/home/rainer/.rednotebook/data/2014-12.txt"
2019-04-04 12:25:42,970 DEBUG    Loading file "/home/rainer/.rednotebook/data/2015-01.txt"
2019-04-04 12:25:42,986 DEBUG    Loading file "/home/rainer/.rednotebook/data/2015-02.txt"
2019-04-04 12:25:42,999 DEBUG    Loading file "/home/rainer/.rednotebook/data/2015-03.txt"
2019-04-04 12:25:43,008 DEBUG    Loading file "/home/rainer/.rednotebook/data/2015-04.txt"
2019-04-04 12:25:43,011 DEBUG    Loading file "/home/rainer/.rednotebook/data/2015-05.txt"
2019-04-04 12:25:43,026 DEBUG    Loading file "/home/rainer/.rednotebook/data/2015-06.txt"
2019-04-04 12:25:43,045 DEBUG    Loading file "/home/rainer/.rednotebook/data/2015-07.txt"
2019-04-04 12:25:43,062 DEBUG    Loading file "/home/rainer/.rednotebook/data/2015-08.txt"
2019-04-04 12:25:43,079 DEBUG    Loading file "/home/rainer/.rednotebook/data/2015-09.txt"
2019-04-04 12:25:43,096 DEBUG    Loading file "/home/rainer/.rednotebook/data/2015-10.txt"
2019-04-04 12:25:43,108 DEBUG    Loading file "/home/rainer/.rednotebook/data/2015-11.txt"
2019-04-04 12:25:43,124 DEBUG    Loading file "/home/rainer/.rednotebook/data/2015-12.txt"
2019-04-04 12:25:43,140 DEBUG    Loading file "/home/rainer/.rednotebook/data/2016-01.txt"
2019-04-04 12:25:43,159 DEBUG    Loading file "/home/rainer/.rednotebook/data/2016-02.txt"
2019-04-04 12:25:43,176 DEBUG    Loading file "/home/rainer/.rednotebook/data/2016-03.txt"
2019-04-04 12:25:43,194 DEBUG    Loading file "/home/rainer/.rednotebook/data/2016-04.txt"
2019-04-04 12:25:43,230 DEBUG    Loading file "/home/rainer/.rednotebook/data/2016-05.txt"
2019-04-04 12:25:43,248 DEBUG    Loading file "/home/rainer/.rednotebook/data/2016-06.txt"
2019-04-04 12:25:43,265 DEBUG    Loading file "/home/rainer/.rednotebook/data/2016-07.txt"
2019-04-04 12:25:43,280 DEBUG    Loading file "/home/rainer/.rednotebook/data/2016-08.txt"
2019-04-04 12:25:43,312 DEBUG    Loading file "/home/rainer/.rednotebook/data/2016-09.txt"
2019-04-04 12:25:43,349 DEBUG    Loading file "/home/rainer/.rednotebook/data/2016-10.txt"
2019-04-04 12:25:43,385 DEBUG    Loading file "/home/rainer/.rednotebook/data/2016-11.txt"
2019-04-04 12:25:43,402 DEBUG    Loading file "/home/rainer/.rednotebook/data/2016-12.txt"
2019-04-04 12:25:43,414 DEBUG    Loading file "/home/rainer/.rednotebook/data/2017-01.txt"
2019-04-04 12:25:43,435 DEBUG    Loading file "/home/rainer/.rednotebook/data/2017-02.txt"
2019-04-04 12:25:43,452 DEBUG    Loading file "/home/rainer/.rednotebook/data/2017-03.txt"
2019-04-04 12:25:43,471 DEBUG    Loading file "/home/rainer/.rednotebook/data/2017-04.txt"
2019-04-04 12:25:43,490 DEBUG    Loading file "/home/rainer/.rednotebook/data/2017-05.txt"
2019-04-04 12:25:43,506 DEBUG    Loading file "/home/rainer/.rednotebook/data/2017-06.txt"
2019-04-04 12:25:43,525 DEBUG    Loading file "/home/rainer/.rednotebook/data/2017-07.txt"
2019-04-04 12:25:43,550 DEBUG    Loading file "/home/rainer/.rednotebook/data/2017-08.txt"
2019-04-04 12:25:43,555 DEBUG    Loading file "/home/rainer/.rednotebook/data/2017-09.txt"
2019-04-04 12:25:43,581 DEBUG    Loading file "/home/rainer/.rednotebook/data/2017-10.txt"
2019-04-04 12:25:43,592 DEBUG    2017-11.CONFLICT_BACKUP1510786427.390858.txt is not a valid month filename
2019-04-04 12:25:43,593 DEBUG    Loading file "/home/rainer/.rednotebook/data/2017-11.txt"
2019-04-04 12:25:43,618 DEBUG    Loading file "/home/rainer/.rednotebook/data/2017-12.txt"
2019-04-04 12:25:43,634 DEBUG    Loading file "/home/rainer/.rednotebook/data/2018-01.txt"
2019-04-04 12:25:43,648 DEBUG    Loading file "/home/rainer/.rednotebook/data/2018-02.txt"
2019-04-04 12:25:43,722 DEBUG    Loading file "/home/rainer/.rednotebook/data/2018-03.txt"
2019-04-04 12:25:43,741 DEBUG    Loading file "/home/rainer/.rednotebook/data/2018-04.txt"
2019-04-04 12:25:43,754 DEBUG    Loading file "/home/rainer/.rednotebook/data/2018-05.txt"
2019-04-04 12:25:43,785 DEBUG    Loading file "/home/rainer/.rednotebook/data/2018-06.txt"
2019-04-04 12:25:43,821 DEBUG    Loading file "/home/rainer/.rednotebook/data/2018-07.txt"
2019-04-04 12:25:43,850 DEBUG    Loading file "/home/rainer/.rednotebook/data/2018-08.txt"
2019-04-04 12:25:43,877 DEBUG    Loading file "/home/rainer/.rednotebook/data/2018-09.txt"
2019-04-04 12:25:43,899 DEBUG    Loading file "/home/rainer/.rednotebook/data/2018-10.txt"
2019-04-04 12:25:43,918 DEBUG    Loading file "/home/rainer/.rednotebook/data/2018-11.txt"
2019-04-04 12:25:43,939 DEBUG    Loading file "/home/rainer/.rednotebook/data/2018-12.txt"
2019-04-04 12:25:43,969 DEBUG    Loading file "/home/rainer/.rednotebook/data/2019-01.txt"
2019-04-04 12:25:43,990 DEBUG    2019-02.CONFLICT_BACKUP1549340467.8491824.txt is not a valid month filename
2019-04-04 12:25:43,990 DEBUG    2019-02.CONFLICT_BACKUP1549340527.5100346.txt is not a valid month filename
2019-04-04 12:25:44,004 DEBUG    Loading file "/home/rainer/.rednotebook/data/2019-02.txt"
2019-04-04 12:25:44,024 DEBUG    2019-03.CONFLICT_BACKUP1553535727.9419005.txt is not a valid month filename
2019-04-04 12:25:44,025 DEBUG    Loading file "/home/rainer/.rednotebook/data/2019-03.txt"
2019-04-04 12:25:44,033 DEBUG    Loading file "/home/rainer/.rednotebook/data/2019-04.txt"
2019-04-04 12:25:44,035 DEBUG    Loading file "/home/rainer/.rednotebook/data/2019-06.txt"
2019-04-04 12:25:44,064 DEBUG    Loading file "/home/rainer/.rednotebook/data/2019-08.txt"
2019-04-04 12:25:44,075 DEBUG    Loading file "/home/rainer/.rednotebook/data/2019-09.txt"
2019-04-04 12:25:44,082 DEBUG    Loading file "/home/rainer/.rednotebook/data/2019-10.txt"
2019-04-04 12:25:44,095 DEBUG    Loading file "/home/rainer/.rednotebook/data/2019-12.txt"
2019-04-04 12:25:44,123 DEBUG    Loading file "/home/rainer/.rednotebook/data/2020-08.txt"
2019-04-04 12:25:44,130 DEBUG    Loading file "/home/rainer/.rednotebook/data/2020-10.txt"
2019-04-04 12:25:44,139 DEBUG    Loading file "/home/rainer/.rednotebook/data/2020-12.txt"
2019-04-04 12:25:44,153 DEBUG    Loading file "/home/rainer/.rednotebook/data/2021-08.txt"
2019-04-04 12:25:44,155 DEBUG    Loading file "/home/rainer/.rednotebook/data/2022-08.txt"
2019-04-04 12:25:44,156 DEBUG    Loading file "/home/rainer/.rednotebook/data/2023-08.txt"
2019-04-04 12:25:44,165 DEBUG    Loading file "/home/rainer/.rednotebook/data/2024-08.txt"
2019-04-04 12:25:44,177 DEBUG    Loading file "/home/rainer/.rednotebook/data/2025-08.txt"
2019-04-04 12:25:44,177 DEBUG    Loading file "/home/rainer/.rednotebook/data/2026-08.txt"
2019-04-04 12:25:44,178 DEBUG    Loading file "/home/rainer/.rednotebook/data/2027-08.txt"
2019-04-04 12:25:44,188 DEBUG    Loading file "/home/rainer/.rednotebook/data/2028-08.txt"
2019-04-04 12:25:44,191 DEBUG    Loading file "/home/rainer/.rednotebook/data/2287-08.txt"
2019-04-04 12:25:44,192 DEBUG    Finished loading files in dir "/home/rainer/.rednotebook/data"
2019-04-04 12:25:44,354 DEBUG    Add mathjax code: False
2019-04-04 12:25:45,264 INFO     Connected Signals: [<Signals.SIGHUP: 1>, <Signals.SIGINT: 2>, <Signals.SIGQUIT: 3>, <Signals.SIGIOT: 6>, <Signals.SIGTERM: 15>, <Signals.SIGTSTP: 20>]
2019-04-04 12:25:45,264 DEBUG    Start took 4.876888990402222 seconds
2019-04-04 12:25:45,264 DEBUG    Trying to enter the gtk main loop
2019-04-04 12:25:46,350 DEBUG    Update the cloud
2019-04-04 12:25:49,563 DEBUG    Cloud updated
2019-04-04 12:25:49,577 INFO     Last backup was made 30 days ago
2019-04-04 12:25:54,346 DEBUG    Add mathjax code: False
2019-04-04 12:25:56,090 DEBUG    Add mathjax code: False
2019-04-04 12:26:41,992 INFO     Wrote file /home/rainer/.rednotebook/data/2019-03.txt
2019-04-04 12:26:41,993 INFO     El contenido ha sido guardado en /home/rainer/.rednotebook/data
2019-04-04 12:26:41,993 INFO     The content has been saved to '/home/rainer/.rednotebook/data'
2019-04-04 12:26:42,419 DEBUG    Update the cloud
2019-04-04 12:26:45,653 DEBUG    Cloud updated
2019-04-04 12:29:13,628 INFO     Wrote file /home/rainer/.rednotebook/data/2019-03.txt
2019-04-04 12:29:13,629 INFO     El contenido ha sido guardado en /home/rainer/.rednotebook/data
2019-04-04 12:29:13,630 INFO     The content has been saved to '/home/rainer/.rednotebook/data'
2019-04-04 12:29:14,062 DEBUG    Update the cloud
2019-04-04 12:29:17,331 DEBUG    Cloud updated
2019-04-04 12:31:59,843 INFO     Wrote file /home/rainer/.rednotebook/data/2019-03.txt
2019-04-04 12:31:59,843 INFO     El contenido ha sido guardado en /home/rainer/.rednotebook/data
2019-04-04 12:31:59,844 INFO     The content has been saved to '/home/rainer/.rednotebook/data'
2019-04-04 12:32:00,289 DEBUG    Update the cloud
2019-04-04 12:32:04,812 DEBUG    Cloud updated
2019-04-04 12:34:27,379 DEBUG    Add mathjax code: False
Traceback (most recent call last):
  File "/usr/bin/rednotebook", line 6, in <module>
    import journal
ImportError: No module named 'journal'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/share/rednotebook/rednotebook/external/spellcheck.py", line 596, in _range_delete
    self.check_range(start, end)
  File "/usr/share/rednotebook/rednotebook/external/spellcheck.py", line 445, in check_range
    self._check_word(word_start, word_end)
  File "/usr/share/rednotebook/rednotebook/external/spellcheck.py", line 623, in _check_word
    word = self._buffer.get_text(start, end, False).strip()
AttributeError: 'NoneType' object has no attribute 'strip'
Traceback (most recent call last):
  File "/usr/bin/rednotebook", line 6, in <module>
    import journal
ImportError: No module named 'journal'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/share/rednotebook/rednotebook/external/spellcheck.py", line 596, in _range_delete
    self.check_range(start, end)
  File "/usr/share/rednotebook/rednotebook/external/spellcheck.py", line 445, in check_range
    self._check_word(word_start, word_end)
  File "/usr/share/rednotebook/rednotebook/external/spellcheck.py", line 623, in _check_word
    word = self._buffer.get_text(start, end, False).strip()
AttributeError: 'NoneType' object has no attribute 'strip'
Traceback (most recent call last):
  File "/usr/bin/rednotebook", line 6, in <module>
    import journal
ImportError: No module named 'journal'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/share/rednotebook/rednotebook/external/spellcheck.py", line 596, in _range_delete
    self.check_range(start, end)
  File "/usr/share/rednotebook/rednotebook/external/spellcheck.py", line 445, in check_range
    self._check_word(word_start, word_end)
  File "/usr/share/rednotebook/rednotebook/external/spellcheck.py", line 623, in _check_word
    word = self._buffer.get_text(start, end, False).strip()
AttributeError: 'NoneType' object has no attribute 'strip'
Traceback (most recent call last):
  File "/usr/bin/rednotebook", line 6, in <module>
    import journal
ImportError: No module named 'journal'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/share/rednotebook/rednotebook/external/spellcheck.py", line 596, in _range_delete
    self.check_range(start, end)
  File "/usr/share/rednotebook/rednotebook/external/spellcheck.py", line 445, in check_range
    self._check_word(word_start, word_end)
  File "/usr/share/rednotebook/rednotebook/external/spellcheck.py", line 623, in _check_word
    word = self._buffer.get_text(start, end, False).strip()
AttributeError: 'NoneType' object has no attribute 'strip'
2019-04-04 12:34:53,788 DEBUG    Main frame destroyed
2019-04-04 12:34:53,802 INFO     Wrote file /home/rainer/.rednotebook/data/2019-04.txt
2019-04-04 12:34:53,826 INFO     Wrote file /home/rainer/.rednotebook/data/2019-03.txt
2019-04-04 12:34:53,828 INFO     El contenido ha sido guardado en /home/rainer/.rednotebook/data
2019-04-04 12:34:53,830 INFO     The content has been saved to '/home/rainer/.rednotebook/data'
2019-04-04 12:34:53,832 INFO     Goodbye!
2019-04-04 12:34:53,931 INFO     Peak memory: 134894216 KiB
```

**Comment by jendrikseipp** [comment]
Thanks for the report. This should be fixed in RedNotebook 2.11.1. Can you confirm that?

**Comment by carnendil** [comment]
I can confirm that the issue appears to be fixed in version 2.11.1.

Thank you.
